ui/src/router/details/logs/RouterLogsView.js

```diff
@@ -77,55 +78,103 @@ export const RouterLogsView = ({ router }) => {
   useEffect(
     () => {
       let urls = {}
+      const {imageBuilderUrl, turingUrl} = appConfig.podLogs.urlTemplates
```

Comment:
It's generally better to initialise constants outside an effect to avoid re-initialisation, especially if the values are constants throughout the lifetime of the app. Helps makes things a bit more readable too.